tests/py/dynamo/conversion/test_upsample.py

```diff
@@ -6,87 +6,172 @@
+            ((2, 2), None, True, (2, 2)),
+            ((2, 2), None, False, (2, 2)),
+            ((2, 2), None, True, (1.5, 1.5)),
+            ((2, 2), None, False, (1.5, 1.5)),
         ]
     )
-    def test_upsample_bilinear_scale_factors(
-        self, _, input_shape, scale_factors, align_corners
+    def test_upsample_bicubic2d(
+        self, input_shape, output_size, align_corners, scale_factors
     ):
         class Upsample(torch.nn.Module):
             def __init__(self):
                 super().__init__()
 
             def forward(self, input):
-                return torch.ops.aten.upsample_bilinear2d.vec(
-                    input,
-                    None,
-                    align_corners,
-                    scale_factors,
+                return torch.ops.aten.upsample_bicubic2d.vec(
+                    input, output_size, align_corners, scale_factors
+                )
+
+        input = [torch.randn([1, 1] + list(input_shape))]
+        self.run_test(Upsample(), input)
+
+    @parameterized.expand(
+        [
+            ((2,), (4,), None),
+            ((2,), (5,), None),
+            ((2,), None, (2,)),
+            ((2,), None, (1.5,)),
+        ]
+    )
+    def test_upsample_nearest1d(self, input_shape, output_size, scale_factors):
+        class Upsample(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                return torch.ops.aten.upsample_nearest1d.vec(
+                    input, output_size, scale_factors
+                )
+
+        input = [torch.randn([1, 1] + list(input_shape))]
+        self.run_test(Upsample(), input)
+
+    @parameterized.expand(
+        [
+            ((2, 2), (4, 4), None),
+            ((2, 2), (5, 5), None),
+            ((2, 2), None, (2, 2)),
+            ((2, 2), None, (1.5, 1.5)),
+        ]
+    )
+    def test_upsample_nearest2d(self, input_shape, output_size, scale_factors):
+        class Upsample(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                return torch.ops.aten.upsample_nearest2d.vec(
+                    input, output_size, scale_factors
+                )
+
+        input = [torch.randn([1, 1] + list(input_shape))]
+        self.run_test(Upsample(), input)
+
+    @parameterized.expand(
+        [
+            ((2, 2, 2), (4, 4, 4), None),
+            ((2, 2, 2), (5, 5, 5), None),
+            ((2, 2, 2), None, (2, 2, 2)),
+            ((2, 2, 2), None, (1.5, 1.5, 1.5)),
+        ]
+    )
+    def test_upsample_nearest3d(self, input_shape, output_size, scale_factors):
+        class Upsample(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+
+            def forward(self, input):
+                return torch.ops.aten.upsample_nearest3d.vec(
+                    input, output_size, scale_factors
                 )
 
```

Comment:
What are the cases in which `torch.ops.aten.upsample*.default` are encountered? I think the above test cases are for
`torch.ops.aten.upsample*.vec`